content/en/docs/shared_services/supportapi/formats/miscellaneous.md

```diff
@@ -14,6 +14,14 @@ Format: `date-time`, with `time-offset` utilizing the `time-numoffset` definitio
 
 Example: `2020-03-11T08:38:11.000+0000`.
 
+### Common Date Format for Responses
+
+All date instances in responses share the same format, as per profile of ISO 8601 at [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339#ref-ISO8601).
```

Comment:
IMO, it's better to use this link `https://tools.ietf.org/html/rfc3339#section-5.6` instead of `https://datatracker.ietf.org/doc/html/rfc3339#ref-ISO8601`, as the paragraph below uses terms from section-5.6 of the RFC and not from the text of the ISO standard .

Just a comment, can be left as is.